pos_partner_location_abstract/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module is base for module pos_partner_location_google_map (Google Map).
```

Comment:
This module implements base functionality for using different map providers in POS.
It allows to select partner location directly on a map and stores selected location in the res.partner model.
It also tries to evaluate partner's address from coordinates and provides a helper function that generates location QR code.
NB: This is a core module which is meant to be used in further customisations.

Reply:
Description is updated